### ISSUE TITLE
Finds movies that are in Plex collections?

### ISSUE THREAD

**Issue by casperse** [JasonHHouse/Gaps#109]
Hi Great project!

This might not be a bug? - But I get a lot of movies as missing even if Plex shows them as being in my current Plex collection and with the correct name in the orig title field

The naming of the folders and files is correct.
The only reason I can think of is that the Gaps tool doesn't use Orig title?
Many movies have a local title in Danish because I wanted the danish description of the Movies and this is the only way to get them

So is this the problem? - Any way to use the orig title field in Plex?
Many people uses Plex in different languages would be a shame if this only worked for english titles?

I guess the folder option would work?

**Comment by JasonHHouse** [comment]
I'm working on a feature to set the language. It'll take a bit of time until I can release it. I have a few things in the pipeline before that.

**Comment by casperse** [comment]
Sounds Wonderful !!
But if the existing tool would just use the "Org title" field instead of the "title" wouldn't language then not be irrelevant and work for everyone?

**Comment by JasonHHouse** [comment]
No, it was changed to handle movies like Pokemon and Dragonball. Those are Japanese movies that people want to view in English. It's not as simple as a flip back and forth.

**Comment by Geek16432** [comment]
Yes, I am seeing this too. I've noticed that it's not consistently handling the case where the title starts with "A","The", or other similar words which Plex might move to the end of the name for listing.
